test/sslapitest.c

```diff
@@ -656,6 +657,194 @@ static int execute_test_large_message(const SSL_METHOD *smeth,
     return testresult;
 }
 
+#ifndef OPENSSL_NO_TLS1_2
+#ifndef OPENSSL_NO_KTLS
```

Comment:
Probably this would be better on one line `#if !defined(OPENSSL_NO_TLS1_2) && !defined(OPENSSL_NO_KTLS)`
Alternatively, if done on two lines then the second one receives a single space indent:
````
#ifndef OPENSSL_NO_TLS1_2
# ifndef OPENSSL_NO_KTLS
````
And then similarly with the endifs:
````
# endif
#endif
````